core/src/main/scala/sttp/openai/requests/completions/chat/ChatRequestBody.scala

```diff
@@ -3,63 +3,74 @@ package sttp.openai.requests.completions.chat
+    *   model's likelihood to talk about new topics.
+    * @param responseFormat
+    *   An object specifying the format that the model must output. Setting to {"type": "json_object"} enables JSON mode, which guarantees
+    *   the message the model generates is valid JSON.
+    * @param seed
+    *   This feature is in Beta. If specified, our system will make a best effort to sample deterministically, such that repeated requests
+    *   with the same seed and parameters should return the same result. Determinism is not guaranteed, and you should refer to the
+    *   system_fingerprint response parameter to monitor changes in the backend.
+    * @param stop
+    *   Up to 4 sequences where the API will stop generating further tokens.
     * @param temperature
     *   What sampling temperature to use, between 0 and 2. Higher values like 0.8 will make the output more random, while lower values like
     *   0.2 will make it more focused and deterministic.
     * @param topP
     *   An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p
     *   probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered.
-    * @param n
-    *   How many chat completion choices to generate for each input message.
-    * @param stop
-    *   Up to 4 sequences where the API will stop generating further tokens.
-    * @param maxTokens
-    *   The maximum number of tokens to generate in the chat completion.
-    * @param presencePenalty
-    *   Number between -2.0 and 2.0. Positive values penalize new tokens based on whether they appear in the text so far, increasing the
-    *   model's likelihood to talk about new topics.
-    * @param frequencyPenalty
-    *   Number between -2.0 and 2.0. Positive values penalize new tokens based on their existing frequency in the text so far, decreasing
-    *   the model's likelihood to repeat the same line verbatim.
-    * @param logitBias
-    *   Modify the likelihood of specified tokens appearing in the completion.
+    * @param tools
+    *   A list of tools the model may call. Currently, only functions are supported as a tool. Use this to provide a list of functions the
+    *   model may generate JSON inputs for.
+    * @param toolChoice
+    *   Controls which (if any) function is called by the model. "none" means the model will not call a function and instead generates a
+    *   message. "auto" means the model can pick between generating a message or calling a function. Specifying a particular function via
+    *   `{"type": "function", "function": {"name": "my_function"}}` forces the model to call that function. "none" is the default when no
+    *   functions are present. "auto" is the default if functions are present.
     * @param user
     *   A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse.
     */
   case class ChatBody(
```

Comment:
** Question ** I'm not sure if sorting arguments alphabetically is a good idea. It may be difficult for current users to find out how they should update their requests with new fields. I would just add new fields to the end of argument list, so that IDE may be more helpful with finding out what should be changed. 
In the future, when new arguments are added, they should also go to the end of the class. Alphabetical order isn't very helpful anyway, right?
Or maybe it's the way OpenAI defines their arguments in the API and we want to have matching models for easier comparison and documentation reading overall?

Reply:
> Or maybe it's the way OpenAI defines their arguments in the API and we want to have matching models for easier comparison and documentation reading overall?

Exactly this one

Reply:
Ok, in that case sorted fields may be justified :)